spec/rails_helper.rb

```diff
@@ -71,6 +72,7 @@
   end
 
   config.before(:suite) do
+    Rails.application.load_tasks
```

Comment:
It would be useful to split these changes out in to another commit, that describes why they are happening.